docs/BasicCommunication/GetFilePath/index.md

```diff
@@ -0,0 +1,81 @@
+## GetFilePath
+
+Type
+: Request
+
+Sender
+: SDL
+
+Purpose
+: Retrieve a file path from the HMI app service provider
```

Comment:
This is being formatted as a table, but incorrectly. Please review https://smartdevicelink.com/en/docs/pull_request/a4c08d3a7aa73f06394196098b108f3c9f8bac09/basiccommunication/getfilepath/ and revise.

Reply:
```suggestion
: Retrieve a file path from the HMI App Service Provider (ASP)
```

Reply:
Please refer to comment https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/139#discussion_r276242113